### ISSUE TITLE
Ground truth preprocessing like T5 prepend method?

### ISSUE THREAD

**Issue by tomatopuree** [google-research/multilingual-t5#113]
Hello,

In the original T5 paper, the summarization ground truths are prepended by the string "summarize: ". Do you follow this training method? If you do, do you prepend a translated version of the summarize string for different languages? I couldn't find a mention of this in the paper.

Thank you

**Comment by tomatopuree** [comment]
Answered my own question. For the folks in the future:

"Note: mT5 was only pre-trained on [mC4](https://huggingface.co/datasets/mc4) excluding any supervised training. Therefore, this model has to be fine-tuned before it is usable on a downstream task, unlike the original T5 model. Since mT5 was pre-trained unsupervisedly, there’s no real advantage to using a task prefix during single-task fine-tuning. If you are doing multi-task fine-tuning, you should use a prefix."
